crates/paimon/src/spec/types.rs

```diff
@@ -124,6 +126,10 @@ impl DataType {
             DataType::Row(v) => v.nullable,
         }
     }
+
+    fn serde_json(&self) -> serde_json::Value {
```

Comment:
Hi, I belive we don't need API like `serde_json`. Users will use `serde_json::to_string(&data_type)` directly